src/server/helpers/merge.js

```diff
@@ -45,6 +45,10 @@ export function getEntityFetchPropertiesByType(entityType) {
 				'languageSet.languages',
 				'workType'
 			];
+		case 'Series':
+			return [
+				'seriesOrderingType'
```

Comment:
is the series `entityType` loaded by default without the need to add it here?

Reply:
Yes it is loaded by default.